wagtail/admin/templates/wagtailadmin/shared/header.html

```diff
@@ -57,5 +57,6 @@ <h1 class="w-header__title">
             {{ extra_actions }}
         </div>
     </div>
-    {% block extra_rows %}{% endblock %}
+    {% block extra_rows %}
+    {% endblock %}
```

Comment:
We probably do not need this change now. Best to remove it unless we need it.

Reply:
https://cs.github.com/wagtail/wagtail?q=block+extra_rows indicates this is still used in some cases. May be good to cross reference with the other bits of related work?